example_text_completion.py

```diff
@@ -56,8 +56,8 @@ def main(
 #        cheese =>""",
     ]
     for i in range(repeat):
-        # Automatically takes profiles, let's skip the cold run and only capture warm runs.
-        if i > 0 and not USE_CUDA and xm.is_master_ordinal():
+        # Automatically takes profiles. Let's skip the cold run and only capture the first warm run.
```

Comment:
Depends on the profile duration, it could capture more runs.

I would suggest probably no changes to the comment.

Reply:
Done, thanks